C-Plus-Plus/search/Unbounded_Binary_Search.cpp

```diff
@@ -0,0 +1,71 @@
+/*
+Given a function ‘int f(unsigned int x)’ which takes a non-negative integer ‘x’ 
+as input and returns an integer as output. The function is monotonically increasing 
+with respect to value of x, i.e., the value of f(x+1) is greater than f(x) for every 
+input x. Find the value ‘n’ where f() becomes positive for the first time.
+Since f() is monotonically increasing, values of f(n+1), f(n+2),… must be positive and 
+values of f(n-2), f(n-3), .. must be negative.
+*/
+
+
+#include<bits/stdc++.h> 
+using namespace std; 
+
+int binarySearch(int low, int high); 
+
+// Let's take an example function 
+// f(x) = x^2 - 10*x - 20 
+int f(int x) { return (x*x - 10*x - 20); } 
```

Comment:
Rewrite this to: Also give proper function name as Don't use single letter to define function as it looks not meaningful.
int f(int x) 
{ 
       return (x*x - 10*x - 20); 
}